automatic/ruby/ruby.nuspec

```diff
@@ -4,24 +4,16 @@
   <metadata>
     <id>ruby</id>
     <version>3.1.0.1</version>
-    <title>Ruby</title>
+    <title>Ruby (Install)</title>
```

Comment:
This is not the Install edition of `ruby`, it should only be named `Ruby` for the meta-package.